planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -319,14 +302,15 @@ BehaviorModuleOutput NormalLaneChange::generateOutput()
     output.path = abort_path_->path;
     insertStopPoint(get_current_lanes(), output.path);
   } else {
-    output.path = getLaneChangePath().path;
+    output.path = status_.lane_change_path.path;
```

Comment:
Just curious. Why it is changed from `getLaneChangePath()`?

Reply:
why do we need to call a function if we can directly access the member variable?